### PR TITLE
Add suggested assetContainer.moveAllFromScene method

### PR DESCRIPTION
**Pull request by TrevorDev**
Helps make it easier to unload a scene to an assetContainer.

See
http://www.html5gamedevs.com/topic/35120-how-to-createadd-assets-directly-to-the-assetcontainer/

Thanks to @hololite on the forum for the code

- [x] Still need to add comments to file
- [x] Add unit test

### CODE REVIEW

File: src/babylon.assetContainer.ts [hunk 1]
```diff
@@ -1,29 +1,109 @@
 module BABYLON {
+    /**
+     * Set of assets to keep when moving a scene into an asset container.
+     */
+    export class KeepAssets {
```
Comment (deltakosh) [inline src/babylon.assetContainer.ts]: Not sure to understand why not lights and other asset types?

File: src/babylon.assetContainer.ts [hunk 2]
```diff
@@ -115,5 +199,50 @@ module BABYLON {
...
+    
+                if (move) {
+                    targetAssets.push(asset);
+                }
+            }
+        }
+
+        /**
+         * Removes all the assets contained in the scene and adds them to the container.
+         * @param keepAssets Set of assets keep in the scene. (default: empty)
```
Comment (deltakosh) [inline src/babylon.assetContainer.ts]: kept ? (I'm sure I'm wrong :))

Comment (TrevorDev) [reply src/babylon.assetContainer.ts]: Will fix: Set of assets to keep in the scene. (default: empty)

File: src/babylon.assetContainer.ts (hunk 1 above)
Comment (TrevorDev) [reply src/babylon.assetContainer.ts]: I missed this. I will add the others and add the missing explicit public to there definitions,

**Pull request merged**